wls-eai-service/src/main/resources/db/dummydata/h2/V12_0__updateWahlbezirkNummerForKomW.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE Wahlbezirk
+SET nummer = 'komw_uwb'
+WHERE id in ('9899cae4-df9a-4ffc-a940-f20cf2280171') OR id in ('9899cae4-df9a-4ffc-a940-f20cf2280172');
```

Comment:
```suggestion
WHERE id in ('9899cae4-df9a-4ffc-a940-f20cf2280171', '9899cae4-df9a-4ffc-a940-f20cf2280172');
```

oder ein `or` aber dann mit `=`